draft-ietf-quic-transport.md

```diff
@@ -1037,8 +1037,17 @@ Servers MUST drop other packets that contain unsupported versions.
 
 Packets with a supported version, or no version field, are matched to a
 connection using the connection ID or - for packets with zero-length connection
-IDs - the address tuple.  If the packet doesn't match an existing connection,
-the server continues below.
+IDs - the address tuple, with the following exception.
```

Comment:
It was also in the old text, but I'm not sure that term `address tuple` is clearly specified. What is meant is of course the source and destination port and IP pair. `tuple` is used elsewhere in two- and three-tuples.